packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/create_saved_objects.test.ts

```diff
@@ -102,7 +102,7 @@ const legacyUrlAliasForObj10 = createLegacyUrlAliasObject(obj10.originId!, obj10
 
 describe('#createSavedObjects', () => {
   let savedObjectsClient: jest.Mocked<SavedObjectsClientContract>;
-  let bulkCreate: typeof savedObjectsClient['bulkCreate'];
+  let bulkCreate: (typeof savedObjectsClient)['bulkCreate'];
```

Comment:
Is this necessary? The format changes modify a lot of core code.

Reply:
Hey @TinaHeiligers the modification to the files happened because of a bump in the version of prettier introduced because of this PR. This is change that would will need to happen eventually if prettier ever gets updated. See [here](https://github.com/prettier/prettier/issues/14990).